kebechet/managers/version/messages.py

```diff
@@ -43,11 +43,13 @@
 Automated version release could not be completed.
 
 Kebechet version manager expects a file with one of the following names: `["setup.py", "__init__.py", "__about__.py",
-"version.py", "app.py", "wsgi.py"]` to contain the line:
+"version.py", "app.py", "wsgi.py", "pyproject.toml"]` to contain the line:
 
 ```
 ...
 __version__ = X.Y.Z
+or
+version = X.Y.Z
```

Comment:
maybe it would be good to specific file like

```suggestion
version = X.Y.Z  (in pyproject.toml)
or
__version__ = X.Y.Z (in the rest of the options)
```